website/docs/static/img/logo.svg

```diff
@@ -1,46 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 17.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="1024px"
-	 height="1024px" viewBox="0 0 1024 1024" enable-background="new 0 0 1024 1024" xml:space="preserve">
-    <g id="Layer_1">
-      <polyline
-        fill="#FFFFFF"
-        points="719.001,851 719.001,639.848 902,533.802 902,745.267 719.001,851"
-      />
-      <polyline
-        fill="#FFFFFF"
-        points="302.082,643.438 122.167,539.135 122.167,747.741 302.082,852.573 302.082,643.438"
-      />
-      <polyline
-        fill="#FFFFFF"
-        points="511.982,275.795 694.939,169.633 512.06,63 328.436,169.987 511.982,275.795"
-      />
-    </g>
-    <g id="Layer_2">
-      <polyline
-        fill="none"
-        stroke="#FFFFFF"
-        stroke-width="80"
-        stroke-miterlimit="10"
-        points="899,287.833 509,513 509,963"
-      />
-      <line
-        fill="none"
-        stroke="#FFFFFF"
-        stroke-width="80"
-        stroke-miterlimit="10"
-        x1="122.167"
-        y1="289"
-        x2="511.5"
-        y2="513"
-      />
-      <polygon
-        fill="none"
-        stroke="#FFFFFF"
-        stroke-width="80"
-        stroke-miterlimit="10"
-        points="121,739.083 510.917,963.042 901,738.333 901,288 511,62 121,289"
-      />
-    </g>
+<svg width="100" height="100" viewBox="0 0 100 100" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We need to find out a way to turn the logo theme sensitive as well

![Screenshot 2021-11-03 at 09 55 37](https://user-images.githubusercontent.com/4838076/140040020-241a92c5-adcc-4f60-9d59-0565a037c226.png)